web/main.go

```diff
@@ -22,11 +22,11 @@ func main() {
 	kubernetesApiHost := flag.String("api-addr", ":8085", "host address of kubernetes public api")
 	templateDir := flag.String("template-dir", "templates", "directory to search for template files")
 	staticDir := flag.String("static-dir", "app/dist", "directory to search for static files")
-	uuid := flag.String("uuid", "", "unqiue Conduit install id")
+	uuid := flag.String("uuid", "", "unique linkerd install id")
```

Comment:
Well... I lowercased it because "id" was already lowercased, and some of the other flags, like "api-addr", use all lowercase as well. But it's definitely a mix ¯\_(ツ)_/¯